src/Core/Components/Button/FluentSplitButton.razor

```diff
@@ -0,0 +1,28 @@
+﻿@namespace Microsoft.FluentUI.AspNetCore.Components
+@using Microsoft.AspNetCore.Components.Rendering;
+@using Microsoft.FluentUI.AspNetCore.Components.Extensions
+@inherits FluentComponentBase
+
+<FluentMenu Class="@ClassValue" Style="@StyleValue" OnClick="@OnMenuClick" OnCheckedChanged="@OnMenuCheckedChanged" split="true">
+    <FluentButton IconStart="@IconStart"
+                  IconOnly="@IconOnly"
+                  Appearance="@(Appearance == ButtonAppearance.Default ? null : Appearance)"
+                  Shape="@Shape"
+                  Size="@Size"
+                  BackgroundColor="@BackgroundColor"
+                  Color="@Color"
+                  OnClick="@OnClick"
```

Comment:
You probably want to have `OnClick="@OnClickHandlerAsync"` or you can remove the `OnClickHandlerAsync` method?

Reply:
OnClickHandlerAsync no needed anymore and removed